licenses: document invite-time license assignment (#25678)
docker/docker.github.io: content/manuals/admin/organization/manage/manage-licenses.md

---
title: Manage license assignment
linkTitle: License assignment
description: Manage product licenses for your organization, including invite-time assignment, revocation, and automatic assignment.
keywords: licenses, organization, members, invite, invitation, Docker Core, Docker Offload, AI Governance, license assignment, docker home
weight: 30
---

Licenses let you selectively choose which of your organization members have
access to supported Docker products. Organization owners can manage active
licenses for their members, or configure licenses to assign automatically when
members access supported Docker products.

> [!TIP]
> To learn more about product licenses, Docker Core seats, and other Docker
> add-ons, see [Docker plans](/manuals/subscription/plans/_index.md),
> or <a href="https://www.docker.com/pricing/contact-sales/" id="dkr_docs_cs_admin_licenses" class="link" rel="noopener">contact sales</a> to purchase licenses.

## Licenses and invites

When you invite someone to your organization, you can assign them a product
license before they accept the invitation. Docker deducts the license when the

> invitee accepts the invitation:

Reviewer: **suggestion:** It sounds a bit ambiguous that a license might be assigned upon sensing an invite. How about rephrasing it...

```suggestion
When you invite someone, you can select a product license to assign when they
accept. Docker doesn't reserve or deduct the license at invite time; assignment
happens on acceptance:
```

Author: Updated to your suggested select-vs-assign / not-reserved-at-invite wording in `740469eb1e`.

Generated by Cursor

- If a license is available when they accept, Docker assigns it to them and the
  number of available licenses decreases by one.
- If no licenses remain when they accept, they still join your organization,

>   but without a license. Docker doesn't notify you or the invitee in this case.

Reviewer: **segession:** I would move the following sentence to a callout block: `Docker doesn't notify you or the invitee in this case` since it's a current limitation of how it's implemented.

Author: Moved the no-notification limitation into a `[!NOTE]` callout in `740469eb1e`.

Generated by Cursor

Confirm your available licenses before sending invitations in bulk. If licenses
run out, invitees can still join your organization but won't have an active

> license.

Reviewer: **issue:** Even if an organization owner confirms available licenses before sending invitations in bulk, there is no guarantee they will be available for each invitee by the time they accept an invite individually.

I would rephrase it a bit...

```suggestion
Licenses aren't reserved for pending invitations, so they must still be
available when each invitee accepts. Monitor availability on the Members page
while invitations are pending.
```

Author: Replaced the bulk-invite confirm wording with your licenses-aren't-reserved / monitor Members page guidance in `740469eb1e`.

Generated by Cursor

### Invite and assign a license

Assigning a license on invite is an alternative to assigning one manually after
they join, or relying on automatic license assignment the first time they use a

> supported product. To assign a license when you invite a member:

Reviewer: **nitpick:** "relying on automatic license assignment the first time they use a supported product" reads as a generally available alternative.

Per the **Automatic license assignment** section on this same page, it's available only for AI Governance licenses, with Docker Core auto-assignment limited to organizations that purchase AI Governance.

Maybe something like `or, where available, relying on automatic license assignment...` adds a qualifier so readers on other plans don't infer they have that option.

Author: Added the `where available` qualifier for automatic license assignment in `740469eb1e`.

Generated by Cursor

1. Sign in to [Docker Home](https://app.docker.com), then choose your
   organization.
1. Select **Members** from the left navigation, then select **Invite**.
1. Select **Emails or usernames**.
1. Enter the email addresses or Docker IDs of the people you want to invite,
   then assign their
   [role](/manuals/enterprise/security/roles-and-permissions/_index.md).

> 1. Under **Licenses (optional)**, select a license that's available to your

Reviewer: **issue:** The box **Licenses (optional)** is multi-select. Therefore...

```suggestion
1. Under **Licenses (optional)**, select one or more licenses that are available to your
```

Author: Updated the step to `select one or more licenses` for multi-select in `740469eb1e`.

Generated by Cursor

   organization.
1. Select **Invite** to send the invite.

For more about sending, resending, and removing invitations, including CSV
file limits, see
[Manage organization members](/manuals/admin/organization/manage/members.md).

### Accept invites

A user invited with license assignment must check their Docker Home

> notifications to accept the invite and assign the license.

Reviewer: **issue:** "must check their Docker Home notifications" that's one option. The invitation email is the primary path.

```suggestion
A user can accept from the link in their invitation email or from their Notifications Center. 
If the selected license is available, Docker assigns it automatically upon acceptance.
```

FYI, Notifications Center is globally available in the header from App or Hub UIs.

Author: Rewrote Accept invites to cover the invitation email or Notifications Center, with automatic assignment on acceptance, in `740469eb1e`.

Generated by Cursor

> notifications to accept the invite and assign the license.

Author: Addressed in `740469eb1e` via Oleg's Accept invites rewrite — acceptance alone triggers assignment; no separate assign action.

Generated by Cursor

## Assign licenses on invite

When you invite someone to your organization, you can assign them a product license before they accept the invitation. This is an alternative to assigning a license manually after they join, or letting automatic license assignment give them a license the first time they use a supported product.

To assign a license when you invite a member:

1. Sign in to [Docker Home](https://app.docker.com), then choose your organization.
1. Select **Members** from the left navigation, then select **Invite**.
1. Select **Emails or usernames**, then enter the email addresses or Docker IDs of the people you want to invite.
1. Assign a role. See [Roles and permissions](/manuals/enterprise/security/roles-and-permissions/_index.md) for details about the access permissions for each role.
1. Optional. Under **Licenses (optional)**, select a license that's available to your organization.

>    <!-- TODO: confirm UI -->

Author: Removed in 555da12ab5. The invite procedure no longer includes the `<!-- TODO: confirm UI -->` marker — the **Licenses (optional)** step and **Invite** button are documented without a placeholder.

Generated by Cursor
